cmd/internal/ecs/ecs.go

```diff
@@ -84,7 +83,10 @@ func init() {
 
 func addAgentToECS(cmd *cobra.Command, args []string) error {
 	// Check for API key
-	key, secret := cfg.GetAPIKeyAndSecret()
+	key, secret, err := cmderr.RequireAPICredentials("The Akita agent must have an API key in order to capture traces.")
+	if err != nil {
+		return err
+	}
 	if key == "" || secret == "" {
 		printer.Errorf("No Akita API key configured. The Akita agent must have an API key in order to capture traces.\n")
 		if env.InDocker() {
```

Comment:
Looks like you didn't remove the old check for empty key or secret.

Reply:
I've removed the old check in: [8d66125](https://github.com/akitasoftware/akita-cli/pull/202/commits/8d661256aba02387c10e5efe17e37b9897feaada)